addon/components/uni-country.js

```diff
@@ -0,0 +1,63 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { isBlank } from '@ember/utils';
+import { inject as service } from '@ember/service';
+import { A } from '@ember/array';
+import layout from '../templates/components/uni-country';
+import Countries from 'ember-cli-countries/enums/countries';
+
+export default Component.extend({
+  layout,
+  i18n: service(),
```

Comment:
I wouldn't add this dependency

Reply:
It's important for us to have the country translations. Also, every project using ember-cli-uniq has i18n

Reply:
Yes, I totally agree 😄  But every project should be able to use its own `translation` addon. This way, we're tying this component to this addon